src/app/login-page/autoregistration/autoregistration.component.ts

```diff
@@ -159,22 +164,21 @@ export class AutoregistrationComponent implements OnInit {
     this.requestService.send(postRequest);
     // Get response
     const response = this.rdbService.buildFromRequestUUID(requestId);
-    // Process response
-    response
-      .pipe(getFirstCompletedRemoteData())
+    response.pipe(getFirstSucceededRemoteData())
       .subscribe(responseRD$ => {
-        if (hasSucceeded(responseRD$.state)) {
-          // Retrieve the token from the response. The token is returned as array of string.
+        if (responseRD$.hasSucceeded) {
```

Comment:
why?

Reply:
Because it has less code, I saw it somewhere else in the DSpace, and I like it more. It does the same thing.